internal/oci/oci_linux.go

```diff
@@ -162,3 +176,40 @@ func metricsToCtrStats(c *Container, m *cgroups.Metrics) *ContainerStats {
 		PIDs:        pids,
 	}
 }
+
+// getTotalInactiveFile returns the value if `total_inactive_file` as integer
+// from `/sys/fs/cgroup/memory/memory.stat`. It returns an error if the file is
+// not parsable.
+func getTotalInactiveFile() (uint64, error) {
+	// no cgroupv2 support right now
+	if isV2, err := cgroups.IsCgroup2UnifiedMode(); err == nil || isV2 {
+		return 0, nil
+	}
```

Comment:
@giuseppe is this the right way to use this function? I cannot omit the check for the `err` because the linter will complain that I not checked that error.

Reply:
yeah I think it is fine.  Thanks for fixing it so quickly!